### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by cmutk**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      <Grid container spacing={2}>
+      {characters&& characters.map((character,index)=>{
+        if(characters.length===index+1){
+          return <Grid key={character.id} ref={lastItemElementRef} item xs={12} sm={3}>
+          <CharacterCard char={character} handleOnClick={handleClickOpen}></CharacterCard>
+        </Grid>
+        } else{
+          return <Grid key={character.id} item xs={12} sm={3}>
+          <CharacterCard char={character} handleOnClick={handleClickOpen}></CharacterCard>
+        </Grid>
```
Comment (orabazu) [inline src/App.js]: no need to conditional rendering you might use `ref={characters.length===index+1 ? lastItemElementRef: undefined}`

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        }
+      }
+      
+      )       
+    }
+      </Grid>  
+      
+      <div><span>Current Page{pageNumber}</span> <span>Total Page Count{pageCount}</span></div>
```
Comment (orabazu) [inline src/App.js]: not visible due to  infinite loader

File: src/App.js (hunk 1 above)
Comment (cmutk) [reply src/App.js]: You are right, thank you ! It's fixed now.

File: src/App.js (hunk 2 above)
Comment (cmutk) [reply src/App.js]: Oops😅😅😅

**Pull request merged**